Extension/src/LanguageServer/extension.ts

```diff
@@ -248,6 +249,11 @@ export async function activate(): Promise<void> {
         clients.timeTelemetryCollector.setFirstFile(activeEditor.document.uri);
         activeDocument = activeEditor.document;
     }
+
+    if (util.extensionContext) {
```

Comment:
Leaving a note so it's not forgotten -- I think we also want to gate on the experimental features setting here.

Reply:
done! Let me know if this is the expected way of doing it.